### PR TITLE
html iframe caption, figures style guide

### PR DESCRIPTION
**Pull request by reveurmichael**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] [local build the book](https://github.com/open-academy/machine-learning/blob/main/CONTRIBUTING.md#build-the-book)
- [ ] review the artifacts from local build
- [ ] [local build the slides](https://github.com/open-academy/machine-learning/blob/main/CONTRIBUTING.md#build-the-slides-optional) if appliable
- [ ] Unit test if appliable

# Checklist:

- [x] My code follows the [style guidelines](https://github.com/open-academy/machine-learning/blob/main/STYLE_GUIDE.md) of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have applied tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by reveurmichael — COMMENTED** [review]
OK

File: open-machine-learning-jupyter-book/_toc.yml [hunk 1]
```diff
@@ -1,192 +0,0 @@
-# Table of contents
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/_toc.yml:1]: why this is removed? I think this is a break change.

File: open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md [hunk 2]
```diff
@@ -0,0 +1,616 @@
+# Machine learning : an overview
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md:1]: Why do we need to create a new ML overview section? ML content already has an introduction section.

File: open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md [hunk 3]
```diff
@@ -0,0 +1,616 @@
...
+## Final Notes
+
+This was about introduction to machine learning basics. Machine learning is a field of computer science and a subfield of AI that is concerned with giving computers the ability to perform some tasks without being programmed explicitly, but rather learning from the data. Machine learning is already transforming industries. It is being used in areas like medicine, banking and finance, consumer electronics, autonomous vehicles, agriculture, etc...
+
+We have also learned about different types of machine learning that are supervised learning, unsupervised learning, semi-supervised learning, self-supervised learning, and reinforcement learning. Also, we saw a typical flow of machine learning projects and the challenges of training learning systems, and how to overcome them.
+
+
+---
+
+## Acknowledgements
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md:614]: Let's use `Acknowledgments`

File: open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md [hunk 4]
```diff
@@ -0,0 +1,616 @@
...
+
+
+## Final Notes
+
+This was about introduction to machine learning basics. Machine learning is a field of computer science and a subfield of AI that is concerned with giving computers the ability to perform some tasks without being programmed explicitly, but rather learning from the data. Machine learning is already transforming industries. It is being used in areas like medicine, banking and finance, consumer electronics, autonomous vehicles, agriculture, etc...
+
+We have also learned about different types of machine learning that are supervised learning, unsupervised learning, semi-supervised learning, self-supervised learning, and reinforcement learning. Also, we saw a typical flow of machine learning projects and the challenges of training learning systems, and how to overcome them.
+
+
+---
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md:612]: remove

File: open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md [hunk 5]
```diff
@@ -0,0 +1,616 @@
...
+An example of confusion matrix
+```
+
+
+From the above confusion matrix:
+
+* `Accuracy = (TP + TN) / (TP + TN + FP + FN) = (71 +36)/(71+36+7+0) = 0.93 or 93%`
+* `Precision = TP / (TP + FP) = 71/(71+7) =0.91 or  91%`
+* `Recall = TP / (TP + FN) = 71/(71+0) = 1, or 100%`
+* `F1 score = 2PR / (P + R) = 2x0.91x1/(0.91+1) = 0.95, or 95%`
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md:550]: These are not code, instead, they are formulation. We should use latex? E.g. $(TP + TN) / (TP + TN + FP + FN) = (71 +36)/(71+36+7+0) = 0.93 or 93%$

File: open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md [hunk 6]
```diff
@@ -0,0 +1,616 @@
...
+
+*Recall = TP / (TP+FN)*
+
+The higher the recall and precision, the better the model is at making accurate predictions but there is a tradeoff between them. Increasing precision will reduce the recall and vice versa. 
+
+A classifier that doesn't have false positives has a precision of 1, and a classifier that doesn't have false negatives has a recall of 1. Ideally, a perfect classifier will have the precision and recall of 1.
+
+We can combine both precision and recall to get another metric called F1 Score. F1 Score is the harmonic mean of precision and recall. 
+
+*F1 Score = 2 x(precision x recall) / (precision + recall)*
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md:532]: To all the formulation, we should use latex.

File: open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md (hunk 5 above)
Comment (reveurmichael) [reply open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md:550]: This section is now removed.

File: open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md (hunk 4 above)
Comment (reveurmichael) [reply open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md:612]: OK

File: open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md (hunk 3 above)
Comment (reveurmichael) [reply open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md:614]: OK

**Pull request merged**